src/main/java/de/ait/secondlife/controllers/CategoryController.java

```diff
@@ -53,6 +53,20 @@ public ResponseEntity<List<CategoryDto>> getAll() {
         return ResponseEntity.ok(service.getAll());
     }
 
+    @Operation(summary = "Get list of the categories, including hidden categories", description = "Accessible to Admin only")
+    @GetMapping("/get-all-plus-hidden")
```

Comment:
I would call this endpoint simply categories/all. I agree that it can be a little confusing due to another get-endpoint in this controller. but here it will be clear from the documentation that the new one is available only to admins and also displays hidden entities.

it just somehow sounds strange from the point of view of human language: there is simply “all”, but it turns out there is also “all plus”

Reply:
decided to rename "get-all-for-admin"